pkg/ccl/crosscluster/logical/logical_replication_writer_processor.go

```diff
@@ -731,6 +742,7 @@ func (lrw *logicalReplicationWriterProcessor) flushBuffer(
 			}
 		}
 	}
+	containsDupes := lrw.dupeCount > initialDupes
```

Comment:
the dupes themselves need to be somewhat recent depending on the map churn; I don't want to claim that this is _the_ dupe count, but rather is the number of duplicate-of-recently-seen or otherwise happened to be detected dupes. basically, it's a subset of the true number, and we don't know the true number